assets/scss/pages/index/_category.scss

```
@@ -0,0 +1,49 @@
+.category-row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: -6px;
+  margin-right: -6px;
+}
+
+.category-col-9 {
+  width: calc(100% / 12 * 9);
+  padding-left: 6px;
+  padding-right: 6px;
+}
+
+.category-col-3 {
+  width: calc(100% / 12 * 3);
+  padding-left: 6px;
+  padding-right: 6px;
+}
+
+.category-card-img {
+  height: 350px;
+
+  transition: transform 0.4s ease;
+  transform: scale(1);
```

```suggestion
  transition: transform 0.4s ease;
```
` transform: scale(1)`這行好像是多的，可以刪除

---

感謝指正，已移除。